container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/docker_kurtosis_backend_enclave_functions.go

```diff
@@ -357,15 +372,33 @@ func (backend *DockerKurtosisBackend) DestroyEnclaves(
 		erroredEnclaveUuids[enclaveUuid] = volumeRemovalErr
 	}
 
+	// Disconnect the containers from the enclave networks being removed
+	networksToDisconnect := map[enclave.EnclaveUUID]string{}
+	for enclaveUuid := range successfulVolumeRemovalEnclaveUuids {
+		networkInfo, found := matchingNetworkInfo[enclaveUuid]
+		if !found {
+			return nil, nil, stacktrace.NewError("Would have attempted to disconnect enclave network '%v' that didn't match the filters", enclaveUuid)
```

Comment:
If this error is hit, likely indicates a bug is in Kurtosis, can we add that? eg. `Attempt was made to disconnect enclave that did not match filters. This is likely a bug in Kurtosis.`